### ISSUE TITLE
Increased serum interleukin 8

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6213]
**Preferred term label:**
Increased serum interleukin-8

**Synonyms**
Increased serum IL-8
Increased serum interleukin 8

**Definition (free text, please give PubMed ID)**
An increased concentration of interleukin-8 in the circulation.

Comment: IL-8 is a chemokine that acts as a chemotactic factor secreted by activated monocytes and macrophages that promotes the directional migration of neutrophils, basophils, and T lymphocytes.
PMID:15831231

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0030782

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:256040, 2/2 individuals, PMID:21129723

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
